File: spring-boot-java-web/src/main/java/com/example/webjsp/SpringBootWebApplication.java
package com.example.webjsp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.cloud.netflix.zuul.EnableZuulProxy;

@SpringBootApplication
@EnableConfigurationProperties
@EnableZuulProxy
public class SpringBootWebApplication extends ServletInitializer {

  public static void main(String[] args) {
    SpringApplication.run(SpringBootWebApplication.class, args);
  }

}

File: spring-boot-kotlin-web/src/main/kotlin/com/example/webjsp/SpringBootWebApplication.kt
package com.example.webjsp

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.cloud.netflix.zuul.EnableZuulProxy

@SpringBootApplication
@EnableZuulProxy
@EnableConfigurationProperties
class SpringBootWebApplication

fun main(args: Array<String>) {
  runApplication<SpringBootWebApplication>(*args)
}
